easybuild/easyconfigs/c/CoordgenLibs/CoordgenLibs-3.0.1-gompi-2023a.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'CMakeMake'
+
+name = 'CoordgenLibs'
+version = '3.0.1'
```

Comment:
any reason not to take the 3.0.2 which supposedly fixes from memory leaks?
https://github.com/schrodinger/coordgenlibs/releases

Reply:
It's already here: https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/c/CoordgenLibs/CoordgenLibs-3.0.2-gompi-2023a.eb